snugget_load.py

```diff
@@ -14,7 +14,7 @@
 
 requiredFields = ['shapefile', 'section', 'subsection']
 # all other fields in snuggetFile are required. The empty string is to deal with Excel's charming habit of putting a blank column after all data in a CSV.
```

Comment:
The second sentence of this comment no longer applies - let's just cut it so it doesn't confuse us later.